### PR TITLE
tsffas

### PR DESCRIPTION
**Pull request by thewolf27**
asfasfafs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/postscripton/laravel-money.svg)](https://packagist.org/packages/postscripton/laravel-money)
 [![License](https://img.shields.io/packagist/l/postscripton/laravel-money)](https://packagist.org/packages/postscripton/laravel-money)
 
-## Introduction
-
```
Comment (thewolf27) [inline README.md:7]: asfasfa

**Review by thewolf27 — COMMENTED** [review]
LGTM

**Review by PostScripton — APPROVED** [review]
Hmm, LGTM

**Comment by PostScripton** [comment]
Of course, we were testing merging pull requests for future contributors

**Pull request closed without merging**